openstack/identity/v3/registered_limits/testing/fixtures.go

```diff
@@ -0,0 +1,219 @@
+package testing
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/registered_limits"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	"github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+// ListOutput provides a single page of List results.
+const ListOutput = `
+{
+    "links": {
+        "self": "http://10.3.150.25/identity/v3/registered_limits",
+        "previous": null,
+        "next": null
+    },
+    "registered_limits": [
+        {
+            "resource_name": "volume",
+            "region_id": RegionOne,
+            "links": {
+                "self": "http://10.3.150.25/identity/v3/registered_limits/25a04c7a065c430590881c646cdcdd58"
+            },
+            "service_id": "9408080f1970482aa0e38bc2d4ea34b7",
+            "id": "25a04c7a065c430590881c646cdcdd58",
+            "default_limit": 11,
+            "description": "Number of volumes for service 9408080f1970482aa0e38bc2d4ea34b7"
+        },
+        {
+            "resource_name": "snapshot",
+            "region_id": "RegionOne",
+            "links": {
+                "self": "http://10.3.150.25/identity/v3/registered_limits/3229b3849f584faea483d6851f7aab05"
+            },
+            "service_id": "9408080f1970482aa0e38bc2d4ea34b7",
+            "id": "3229b3849f584faea483d6851f7aab05",
+            "default_limit": 5,
+            "description": null
+        }
+    ]
+}
+`
+
+// GetOutput provides a Get result.
+const GetOutput = `
+{
+    "registered_limit": {
+        {
```

Comment:
Here you're have one too many level of curly braces, that's why you're seeing the `"invalid character '{' looking for beginning of object key string"` error [in the unit tests](https://github.com/gophercloud/gophercloud/actions/runs/4978055778/jobs/8907869530?pr=2616#step:6:144).